src/components/BottomMenu.tsx

```diff
@@ -82,7 +82,9 @@ export default function BottomMenu({
           <MenuItem
             label={formatMessage(noticeAndDoNotSellMessages.doNotSellLabel)}
             type="button"
-            onClick={() => handleSetViewState(ViewState.CompleteOptions)}
+            onClick={() => {
+              handleSetViewState(ViewState.CompleteOptions);
+            }}
```

Comment:
```suggestion
            onClick={() => handleSetViewState(ViewState.CompleteOptions)}
```

Reverts to make this a non-change. It doesn't actually matter but technically a different return type here